index.html

```diff
@@ -272,134 +315,188 @@ <h2>Glossary</h2>
+<p><dfn class="lint-ignore export">Natural Language</dfn> (sometimes just <em>language</em>). Refers to the spoken, written, or signed communications used by human beings. See also [=localizable text=] and [=syntactic content=].</p>
+
+<p><dfn class="lint-ignore export">Non-linguistic Field</dfn>. Any element of a data structure not intended for the storage or interchange of natural language textual data. This includes non-string data types, such as booleans, numbers, dates, and so forth. It also includes strings, such as program or protocol internal identifiers. This document uses the term <em>field</em> as a short hand for this concept.</p>
 
-<p><dfn data-lt="non-linguistic field|field|non-linguistic fields|fields|Non-linguistic Field" id="def_non_linguistic_field" class="lint-ignore export">Non-linguistic Field</dfn>. Any element of a data structure not intended for the storage or interchange of natural language textual data. This includes non-string data types, such as booleans, numbers, dates, and so forth. It also includes strings, such as program or protocol internal identifiers. This document uses the term <em>field</em> as a short hand for this concept.</p>
+<p><dfn class="lint-ignore export" data-lt="normalisation|Unicode normalization">Normalization</dfn>. The process of removing alternate representations of equivalent sequences from textual data, to convert the data into a form that can be binary-compared for equivalence.</p>
+
+<p><dfn class="lint-ignore export" data-lt="unicode normalization form c|nfc">Normalization Form C</dfn> (<strong>NFC</strong>). <a href="https://www.unicode.org/glossary/#normalization_form_c">Unicode definition</a>: <q cite="https://www.unicode.org/glossary/#normalization_form_c">A normalization form that erases any canonical differences, and generally produces a <strong>composed</strong> result. For example, a + umlaut is 
+converted to ä in this form. This form most closely matches legacy 
+usage. The formal definition is D120 in <a href="http://www.unicode.org/versions/latest/ch03.pdf#G49537">
+Section 3.11, Normalization Forms</a>.</q> See also [=normalization=].</p>
+
+<p><dfn class="lint-ignore export" data-lt="unicode normalization form d|nfd">Normalization Form D</dfn> (<strong>NFD</strong>). <a href="https://www.unicode.org/glossary/#normalization_form_c">Unicode definition</a>: <q cite="https://www.unicode.org/glossary/#normalization_form_d">A normalization form that erases any canonical differences, and produces a <strong>decomposed</strong> result. For example, ä is converted to a + umlaut in this form. This form is most often used in internal processing, such as in collation. The formal definition is D118 in <a href="http://www.unicode.org/versions/latest/ch03.pdf#G49537"> Section 3.11, Normalization Forms</a>.</q> See also [=normalization=].</p>
```

Comment:
Add the K forms?

Reply:
Haven't had time to get to them yet.  Feel free to suggest a couple of definitions.

Reply:
Leaving this open for future update.